### ISSUE TITLE
On build, Command failed: dotnet build

### ISSUE THREAD

**Issue by Osuna0102** [microsoft/BotFramework-Composer#9371]
## Describe the bug

I downloaded the Bot Composer using the guide https://docs.microsoft.com/en-us/composer/install-composer?tabs=windows#prerequisites-for-installing-composer-as-a-desktop-application
However, I cant create a new bot using C#. Using Node I can with no problems, but I need it with C#. 
I followed almost every guide that was on here, including this steps:
https://github.com/microsoft/BotFramework-Composer/issues/7876#issuecomment-881082316

however nothing was wrong. I updated drivers, reinstalled Composer and NET CORE but no results.

Bot Composer: v2.1.2
.NET Core SDK: v3.1.422

## To Reproduce

1. Create an empty bot.
![image](https://user-images.githubusercontent.com/55521810/188519241-90c46537-3658-42f3-b230-1aabf67c451b.png)

2. Click create after setting it up.
![image](https://user-images.githubusercontent.com/55521810/188519281-d625ff9e-ea5b-4fa1-a74e-a4ebeae38eb5.png)

4. Loads for a while

![image](https://user-images.githubusercontent.com/55521810/188519320-fd984ad7-c32a-4362-b12a-8b6e7fc0e135.png)

5. See error
![image](https://user-images.githubusercontent.com/55521810/188519645-539eec6a-3584-4db1-abb7-c36257b17742.png)

## Expected behavior

Create a simple, empty bot.

Thanks in advance to anyone who helps me.

**Comment by anishprasad01** [comment]
Hi @Osuna0102,

When you created a new C# bot outside of Composer using yeoman, what was the output of the `dotnet build` command? Did you get any error messages? If not, were you able to then run the bot? If the latter, what happened when you tried to run it?

**Comment by Osuna0102** [comment]
Hi, @anishprasad01 thanks for the reply, im a little noob on things so please bear with me.
![image](https://user-images.githubusercontent.com/55521810/188777848-dfbc9de9-dc17-4cb9-baa2-a2fa0fca7f60.png)
I got this, which is in spanish, and the translation goes like this: "Specify the project or solution file to be used as this folder contains more than one project or solution file."

**Comment by anishprasad01** [comment]
The error message seems to be suggesting you have multiple `csproj` files within the same folder. Have you ensured there is only one bot's files within the folder? What are the contents of the folder?

**Comment by Osuna0102** [comment]
![image](https://user-images.githubusercontent.com/55521810/188960763-d8d3aa1e-3dc5-45a4-b74a-4b7824170341.png)
This is how the folder looks. I dont see more than 1 csproj or sln files in it.
I re-created the bot in case I messed up before, but I got the same error.

**Comment by anishprasad01** [comment]
Delete the `.botproj` file and rebuild. This is an older template, but this might help up find out why `dotnet build` is failing.

**Comment by Osuna0102** [comment]
![image](https://user-images.githubusercontent.com/55521810/188996357-305db41a-8708-47fb-9bb3-742f5e69b12c.png)
It says: Determinating which proyects to restaurate:
Error: Local origin of ProgramFiles/Microsoft SDKs/NuGetPacakges does not exist
Error compiling
Error: Local origin of ProgramFiles/Microsoft SDKs/NuGetPacakges does not exist

Is that usefull?

**Comment by Osuna0102** [comment]
I solved it. Thank you very much for your help. 

I searched on google the error stated above: "Local origin of ProgramFiles/Microsoft SDKs/NuGetPacakges does not exist"
Used this guide https://stackoverflow.com/questions/56558843/local-source-of-nuget-packages-doesnt-exist
and just used dotnet new nugetconfig to build repair it.

If anyone else is running this problem, just run on cmd "dotnet new nugetconfig" and it should be fixed
